src/psyclone/transformations.py

```diff
@@ -556,21 +544,18 @@ def __init__(self):
     def __str__(self):
         return "Adds an 'OpenACC loop' directive to a loop"
 
-    def _directive(self, parent, children, collapse=None):
+    def _directive(self, children, collapse=None):
         '''
         Creates the ACCLoopDirective needed by this sub-class of
         transformation.
 
-        :param parent: the parent Node of the new directive Node.
-        :type parent: :py:class:`psyclone.psyir.nodes.Node`
         :param children: list of child nodes of the new directive Node.
         :type children: list of :py:class:`psyclone.psyir.nodes.Node`
         :param int collapse: number of nested loops to collapse or None if \
                              no collapse attribute is required.
         '''
         from psyclone.psyGen import ACCLoopDirective
```

Comment:
Please could you move this import to the top.

Reply:
Done